Mystery.py

```diff
@@ -65,14 +65,17 @@ def main():
         path = getattr(args, f'p{player}')
         if path:
             if path not in weights_cache:
-                weights_cache[path] = get_weights(path)
+                try:
+                    weights_cache[path] = get_weights(path)
+                except:
+                    raise ValueError(f"File {path} is destroyed. Please fix your yaml.")
             print(f"P{player} Weights: {path} >> {weights_cache[path]['description']}")
 
     erargs = parse_arguments(['--multi', str(args.multi)])
     erargs.seed = seed
     erargs.names = args.names
     erargs.create_spoiler = args.create_spoiler
-    erargs.race = True
+    erargs.race = False
```

Comment:
Also suggested, make this hard-coded line into a command line argument that defaults to True if not present.